scripts/yocto-build-and-test.sh

```diff
@@ -213,18 +213,18 @@ PREFERRED_PROVIDER_virtual/mender-client-native = "mender-native"
 EOF
     fi
 
-    # Assuming sumo or newer
     cat >> $BUILDDIR/conf/local.conf <<EOF
-# MEN-2948: Renamed mender recipe -> mender-client
-# But the "mender" reference has to be kept for backwards compatibility
-# with 2.1.x, 2.2.x, and 2.3.x
 EXTERNALSRC${sep}pn-mender = "$WORKSPACE/go${BUILD_CPP_CLIENT_PATH:-}"
 EXTERNALSRC${sep}pn-mender-native = "$WORKSPACE/go${BUILD_CPP_CLIENT_PATH:-}"
 EXTERNALSRC${sep}pn-mender-client = "$WORKSPACE/go"
 EXTERNALSRC${sep}pn-mender-client-native = "$WORKSPACE/go"
 EXTERNALSRC${sep}pn-mender-artifact = "$WORKSPACE/go"
 EXTERNALSRC${sep}pn-mender-artifact-native = "$WORKSPACE/go"
 EXTERNALSRC${sep}pn-mender-connect= "$WORKSPACE/go"
+
+# When using externalsrc from CI, we still want to apply patches
+SRCTREECOVEREDTASKS${sep}remove = "do_patch"
```

Comment:
Hoho, evil! But also clever.. I can't think of a way to "break" your idea! 🙂

As long as it's contained to our script, this may actually work.

Reply:
I think it is okey. As you said it is contained to the CI script, and I put it very close to the `EXTERNALSRC` to make it clear that it will only affect these (which is our software).